.gitlab-ci.yml

```diff
@@ -1009,6 +1009,15 @@ stages:
     JANE_DIFF_PATTERN: '.*/debops.roundcube/.*'
     JANE_LOG_PATTERN: '\[debops\.roundcube\]'
 
+'roundcube/mariadb role':
+  <<: *test_role_2rd_deps
```

Comment:
That would need to be `test_role_2nd_deps`. Maybe I need to change that to a simplier version...